less/_components/collapsible-menu.less

```diff
@@ -0,0 +1,61 @@
+
+.collapsible-menu {
+    margin-top:10px;
+
+    a, a:hover {
+        color: @main-sidebar-color;
+    }
+
+    .collapsible-menu-item {
+      background-color: @main-sidebar-bg;
+      padding:10px 0 10px;
+
+      &.active {
+        background-color: @main-sidebar-bg;
+        a {
+          color:@main-sidebar-color;
+        }
+        &:hover {
+          background-color: darken(@main-sidebar-bg, 15%);
+        }
+      }
+      &:hover:not(.active) {
+        background-color: darken(@main-sidebar-bg, 15%);
+      }
+    }
+
+    .collapsible-menu-item:not(:last-child) {
+        border-bottom: solid @main-sidebar-list-border 1px;
+    }
+
+    .collapsible-menu-sub-item {
+      background-color:@white;
+      a {
+        color:  @main-sidebar-bg;
+        &:hover {
+          font-weight:bold;
+        }
+      }
+    }
+    .sidebar-sub-item {
```

Comment:
You forgot ths one.

Reply:
.mobile-sidebar => .collapsible-menu
.sidebar-item => .collapsible-menu-item
.sidebar-sub-items => .collapsible-menu-sub-items
.sidebar-text => .collapsible-menu-text
.sidebar-sub-item =>  .collapsible-menu-sub-item

Reply:
Updated